src/Spice86/Spice86DependencyInjection.cs

```diff
@@ -282,7 +283,7 @@ public Spice86DependencyInjection(ILoggerService loggerService, Configuration co
                 breakpointsViewModel, pauseHandler, messenger,
                 uiThreadDispatcher, textClipboard,
                 hostStorageProvider, structureViewModelFactory,
-                canCloseTab: false, startAddress: stack.PhysicalAddress);
+                canCloseTab: false, startAddress: ConvertUtils.ToHex32(stack.PhysicalAddress));
```

Comment:
This is not needed, it'll jump to the stack address anyway
```suggestion
                canCloseTab: false);
```